internal/storage/http_test.go

```diff
@@ -17,21 +17,21 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/rand"
 
-	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+	rukpakv1alpha2 "github.com/operator-framework/rukpak/api/v1alpha2"
 	"github.com/operator-framework/rukpak/internal/util"
 )
 
 var _ = Describe("HTTP", func() {
 	var (
-		ctx        context.Context
-		bundle     *rukpakv1alpha1.Bundle
-		testFS     fs.FS
-		localStore *LocalDirectory
-		server     *httptest.Server
+		ctx              context.Context
+		bundledeployment *rukpakv1alpha2.BundleDeployment
```

Comment:
Global: please make sure to use camelCase. In other words: `bundleDeployment` instead of `bundledeployment`